app/assets/stylesheets/user_show.scss

```diff
@@ -17,12 +17,16 @@
    right:10px;
   }
 
+  #show-score {
```

Comment:
Avoid using id selectors<br>Line should be indented 0 spaces, but was indented 2 spaces